### PR TITLE
Fix memory leaks and update README.md

### PR DESCRIPTION
**Pull request by faf0**
- Fix memory leaks of UNIX version
- Added dependency installation step to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ A color temperature setting library and CLI that operates in a similar way to f.
 The command line app automatically determines your location using GeoIP and adjusts the color temperature depending on time.
 
 ```sh
+$ go get -d github.com/d4l3k/go-sct/sct
```
Comment (d4l3k) [inline README.md]: This two lines should probably just be 

``` bash
go get -u github.com/d4l3k/go-sct/sct
```

**Comment by d4l3k** [comment]
Thanks for fixing my mistakes! I forgot about those since they're not in the original sct.

File: README.md (hunk 1 above)
Comment (faf0) [inline README.md]: That's true. I will push a commit.

**Comment by faf0** [comment]
No problem. I noticed that the original sct never freed some memory when I ran valgrind on my fork https://github.com/faf0/sct/ .

File: README.md (hunk 1 above)
Comment (faf0) [inline README.md]: The commit is here https://github.com/faf0/go-sct/commit/f30b03ba0527a32a099a51297d0ef49dc0478b35

**Comment by d4l3k** [comment]
Looks good to me.

**Pull request merged**